src/main/resources/findologic_20.xsd

```diff
@@ -47,6 +47,28 @@
             <xs:element type="bonusesType" name="bonuses" minOccurs="0" maxOccurs="1"/>
             <xs:element type="sortsType" name="sorts" minOccurs="0" maxOccurs="1"/>
             <xs:element type="visibilitiesType" name="visibilities" minOccurs="0" maxOccurs="1"/>
+            <xs:element type="variantsType" name="variants" minOccurs="0" maxOccurs="1"/>
+        </xs:all>
+        <xs:attribute type="nonEmptyString" name="id" use="required"/>
+    </xs:complexType>
+
+    <xs:complexType name="variantsType">
+        <xs:sequence>
+            <xs:element type="variantType" name="variant" minOccurs="1" maxOccurs="unbounded"/>
```

Comment:
Do we really need this property `minOccurs="1"`?

Reply:
You mean, because the default is already `1`? I would still keep it in like that. It just helps to get the value immediately without knowing the default

Reply:
No, I meant why it's not `0`!?

Reply:
>  Do you have any concerns?
Not really, it's up to you and to the reviewer.

Reply:
:thinking: An empty `<variants/>` tag doesn't make sense logically, because one could just omit if it a product has no variants. For someone who implements an XML export, it might be easier to allow this element to be empty, though, because their implementation could look like this:

1.  Write elements for parent product information.
2.  Start `<variants>` tag.
3.  Loop over variant products and add their tags.
4.  Close the `<variants>` tag.

If a product has no variants, this simple implementation would be invalid according to current rules, requiring the implementer to add an extra check before starting the `<variants>` tag to ensure that at least one variant exists.

On our side, things are slightly easier  this way, because we can always assume that `<variants>` only exists if it has content. If we design the schema for our convenience, we only benefit from this convenience once. If we design the schema for the export implementer's convenience, each of them benefits.

Reply:
Ok, then let's allow an empty variants tag